<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogConfigChambre</class>
 <widget class="QDialog" name="DialogConfigChambre">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1041</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/logo.png</normaloff>:/images/logo.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Caract√©ristiques</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" rowspan="2">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>Chambres</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Type de chambre :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBoxTypeChambre">
              <property name="sizeAdjustPolicy">
               <enum>QComboBox::AdjustToContents</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="checkBoxTypeChDispo">
            <property name="maximumSize">
             <size>
              <width>110</width>
              <height>23</height>
             </size>
            </property>
            <property name="text">
             <string>disponible</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="btn_ajouterUnTypeChambre">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>95</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Ajouter un type</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLabel" name="labelNbChambres">
            <property name="text">
             <string>00 chambres</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="3">
           <widget class="QTableView" name="tableViewChambres">
            <property name="minimumSize">
             <size>
              <width>240</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <attribute name="horizontalHeaderMinimumSectionSize">
             <number>110</number>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderCascadingSectionResizes">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderDefaultSectionSize">
             <number>21</number>
            </attribute>
            <attribute name="verticalHeaderDefaultSectionSize">
             <number>21</number>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderCascadingSectionResizes">
             <bool>true</bool>
            </attribute>
            <attribute name="horizontalHeaderMinimumSectionSize">
             <number>110</number>
            </attribute>
           </widget>
          </item>
          <item row="3" column="0" colspan="3">
           <widget class="QSplitter" name="splitter">
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QPushButton" name="btn_supprimer">
             <property name="maximumSize">
              <size>
               <width>75</width>
               <height>23</height>
              </size>
             </property>
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <property name="text">
              <string>Supprimer</string>
             </property>
            </widget>
            <widget class="QPushButton" name="btn_mettreHS">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>23</height>
              </size>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string>Mettre HS</string>
             </property>
            </widget>
            <widget class="QPushButton" name="btn_modifier">
             <property name="maximumSize">
              <size>
               <width>75</width>
               <height>23</height>
              </size>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string>Modifiier</string>
             </property>
            </widget>
            <widget class="QPushButton" name="btn_ajouter">
             <property name="maximumSize">
              <size>
               <width>75</width>
               <height>23</height>
              </size>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string>Ajouter</string>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>355</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>425</width>
           <height>111</height>
          </size>
         </property>
         <property name="title">
          <string>Configurer les types de client</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Type de client :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBoxTypeClient">
              <property name="sizeAdjustPolicy">
               <enum>QComboBox::AdjustToContents</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="btn_ajouterUnTypeClient">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>95</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Ajouter un type</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Profile de client :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBoxProfileClient">
              <property name="sizeAdjustPolicy">
               <enum>QComboBox::AdjustToContents</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="btn_ajouterUnTypeProfile">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>95</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Ajouter un profile</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Tarifs des chambres</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QTableWidget" name="tableWidgetTarifsChambres">
            <attribute name="verticalHeaderDefaultSectionSize">
             <number>20</number>
            </attribute>
            <attribute name="verticalHeaderDefaultSectionSize">
             <number>20</number>
            </attribute>
            <row>
             <property name="text">
              <string>Adolescent</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string>Chambre ventil√©e</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="btn_appliquer">
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>23</height>
             </size>
            </property>
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="text">
             <string>Appliquer</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="btn_fermer">
     <property name="maximumSize">
      <size>
       <width>75</width>
       <height>23</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Fermer</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
